v1/cmd/create/create.go

```
@@ -0,0 +1,146 @@
+		return
+	}
+	fmt.Printf("Java 程序输出:\n%s\n", string(utf8Bytes))
+}
+
+func init() {
+	root.RootCmd.AddCommand(createCmd)
+
+	createCmd.Flags().StringP("projectPath", "p", "", "项目路径 (必填)")
+	createCmd.Flags().StringP("applicationName", "a", "", "应用名称 (必填)")
+
+	createCmd.MarkFlagRequired("projectPath")
```

golangci-lint

Error return value of `createCmd.MarkFlagRequired` is not checked

(errcheck)